source/documentation/administration_guide/topics/Host_Power_Management_settings_explained.adoc

```diff
@@ -10,7 +10,7 @@ The *Power Management* settings table contains the information required on the *
 |===
 |Field Name |Description
 |*Enable Power Management* |Enables power management on the host. Select this check box to enable the rest of the fields in the *Power Management* tab.
-|*Kdump integration* |Prevents the host from fencing while performing a kernel crash dump, so that the crash dump is not interrupted. In {enterprise-linux} 7.1 and later, kdump is available by default. If kdump is available on the host, but its configuration is not valid (the kdump service cannot be started), enabling *Kdump integration* will cause the host (re)installation to fail. If this is the case, see xref:fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration].
+|*Kdump integration* |Prevents the host from fencing while performing a kernel crash dump, so that the crash dump is not interrupted. In {enterprise-linux} 7.1 and later, kdump is available by default. If kdump is available on the host, but its configuration is not valid (the kdump service cannot be started), enabling *Kdump integration* will cause the host (re)installation to fail. If this is the case, see xref:fence_kdump_Advanced_Configuration[fence_kdump Advanced Configuration]. If you enable kdump, you must reinstall the host in order to apply the correct fencing setting.
```

Comment:
I'd change the sentence to:

If you enable Kdump Integration or Disable Kdump Integration, you must perform Reinstall operation on the host in order to apply this option to the host

Reply:
done